src/main/java/ch/heigvd/dai/commands/Root.java

```diff
@@ -0,0 +1,83 @@
+package ch.heigvd.dai.commands;
+
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        description = "A command-line interface (CLI) tool that allows users to generate QR codes by providing text or URLs as input.",
+        version = "1.0.0",
+        scope = CommandLine.ScopeType.INHERIT,
+        mixinStandardHelpOptions = true)
+
+public class Root implements Callable<Integer>{
+
+    public enum AvailableOutputFormat {

```

Comment:
I think it look better in the CLI if all enums implemented `toString` to have lowercase values.